QuickGuidePanels/QuickGuidePanels.js

```diff
@@ -1,80 +1,448 @@
+import {Cell, Column, Row} from '@enact/ui/Layout';
+import Changeable from '@enact/ui/Changeable';
+import ForwardRef from '@enact/ui/ForwardRef';
+import ViewManager from '@enact/ui/ViewManager';
+import IString from 'ilib/lib/IString';
 import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
 
-import WizardPanels from '../WizardPanels';
+import $L from '../internal/$L';
+import Button from '../Button';
+import {BasicArranger, CancelDecorator, NavigationButton} from '../internal/Panels';
+import PanelsRouter from '../internal/Panels/PanelsRouter';
+import Skinnable from '../Skinnable';
+import Steps from '../Steps';
+
+import css from './QuickGuidePanels.module.less';
 
 /**
  * A QuickGuidePaenls that has steps with corresponding panels and panels have full screen size content.
  *
  * @example
  * 	<QuickGuidePanels>
  *		<QuickGuidePanels.Panel>
- *			<Scroller>
- *				lorem ipsum ...
- *			</Scroller>
- *			<footer>
- *				<Button>OK</Button>
- *				<Button>Cancel</Button>
- *			</footer>
+ *			QuickGuidePanelsContent
  *		</QuickGuidePanels.Panel>
  *	</QuickGuidePanels>
  *
+ * @class QuickGuidePanelsBase
+ * @memberof sandstone/QuickGuidePanels
+ * @ui
+ * @public
+ */
+const QuickGuidePanelsBase = kind({
+	name: 'QuickGuidePanels',
+
+	propTypes: /** @lends sandstone/QuickGuidePanels.QuickGuidePanelsBase.prototype */ {
+		'aria-label': PropTypes.string,
+
+		/**
+		 * Hint string read when focusing the close button.
+		 *
+		 * @type {String}
+		 * @default 'Exit Quick Guide'
+		 * @private
+		 */
+		closeButtonAriaLabel: PropTypes.string,
+
+		/**
+		 * Obtains a reference to the root node.
+		 *
+		 * @type {Function|Object}
+		 * @public
+		 */
+		componentRef: EnactPropTypes.ref,
+
+		/**
+		 * The current step.
+		 *
+		 * This is 1-based, not 0-based; as in the first step is `1`. If omitted, this will equal
+		 * the currently selected panel.
+		 *
+		 * @type {Number}
+		 * @public
+		 */
+		current: PropTypes.number,
+
+		/**
+		* The currently selected panel.
+		*
+		* @type {Number}
+		* @default 0
+		* @private
+		*/
+		index: PropTypes.number,
+
+		/**
+		 * The button to use in place of the standard next button.
+		 *
+		 * This prop accepts a component (e.g. `Button`), a component instance, or a boolean value.
+		 *
+		 * If `false`, the button will not show. If set to a component, or `true`, the button will
+		 * show. This will override the setting of `nextButtonVisibility`.
+		 *
+ 		 * Example:
+ 		 * ```
+		 * nextButton={<Button icon="closex" aria-label="Quit">Close</Button>}
+		 * ```
+		 *
+		 * @type {Boolean|Component}
+		 * @private
+		 */
+		nextButton: PropTypes.oneOfType([PropTypes.bool, EnactPropTypes.componentOverride]),
+
+		/**
+		 * Specifies when and how to show `nextButton` on QuickGuidePanel.
+		 *
+		 * * `'auto'` will display the `nextButton` on every `QuickGuidePanel.Panel` except the last
```

Comment:
@vJIYEv, It seems that QuickGuidePanel.Panel does not make sense. Please check it.

Reply:
I checked and changed.
and also changed WizardPanel.Panel to WizardPanels.Panel